examples/mipmap/main.rs

```diff
@@ -54,6 +56,27 @@ fn create_texels(size: usize, cx: f32, cy: f32) -> Vec<u8> {
         .collect()
 }
 
+struct QuerySets {
```

Comment:
could we add a line to our `examples/README.md` showing that this API is demonstrated in the mipmap example?